rpcs3/Emu/Cell/lv2/sys_spu.cpp

```diff
@@ -1340,9 +1370,9 @@ error_code sys_raw_spu_create_interrupt_tag(u32 id, u32 class_id, u32 hwthread,
 	{
 		std::shared_ptr<lv2_int_tag> result;
 
-		auto thread = idm::check_unlocked<RawSPUThread>(id);
+		auto thread = idm::check_unlocked<named_thread<spu_thread>>(spu_thread::find_raw_spu(id));
```

Comment:
Instead of allocating rawspu ids on threaded spu space, its possible to save a table of 5 RAW threads pointers. like g_raw_spu_threads table. this is convenient as their ids can correspond with their index on such table. (not using idm)

Reply:
It'll be hard to manage in thread-safe manner. Also I used this as an intermediate solution.